### PR TITLE
✨  collection time frame select + refactor

### LINKED ISSUE #4632: Collection time frame select
image

It does not have active state, so you dont know what time frame is selected. 
should have active state like here: 
- #4603 

the gap between buttons should be -1px because like this the stroke double stack.

### PR DESCRIPTION
**Pull request by petersopko**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [x] Refactoring

## Context

- [x] Closes #4632 
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image
image

### CODE REVIEW

**Comment by exezbcz** [comment]
yup, that is it.

File: components/landing/topCollections/TopCollections.vue [hunk 1]
```diff
@@ -3,35 +3,14 @@
...
-            :label="`${$t('topCollections.timeFrames.all')}`"
-            @click.native="setTimeRange('All')" />
-        </div>
+        class="top-collection-controls buttons is-align-items-flex-start pt-2">
+        <NeoButton
+          v-for="{ value, label } in timeRanges"
+          :key="value"
+          class="control column p-0 has-fixed-width px-4 mobile-padding"
+          :active="state.timeRange === value"
+          :label="`${$t(`topCollections.timeFrames.${label}`)}`"
```
Comment (vikiival) [inline components/landing/topCollections/TopCollections.vue:12]: maybe translation keeys like: `week, month, quarter, all` ?

**Comment by vikiival** [comment]
pay 40 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $40 @ 26.65 USD/KSM ~ 1.501 $KSM
🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)
🔗 [0xbf6231e03c43291b528d4ceb583d1ec899eb5b543cfc099b475a83c2862b6b25](https://kusama.subscan.io/extrinsic/0xbf6231e03c43291b528d4ceb583d1ec899eb5b543cfc099b475a83c2862b6b25)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**